s3repo/repoinfo.py

```diff
@@ -48,10 +48,10 @@ def __init__(self, path, supported_repos):
         self.dist_version = path_list[3]
 
     def __str__(self):
-        return '/'.join(self.repo_kind,
+        return '/'.join([self.repo_kind,
                         self.tarantool_series,
                         self.dist,
-                        self.dist_version)
+                        self.dist_version])
```

Comment:
Nit: you can use a tuple instead of a list here. It is immutable and so may leave AOT compiler more opportunities for optimizations.

Don't know, whether it really matters. Feel free to close without actions.

Reply:
Updated.